minecraft-server/start-minecraft.sh

```diff
@@ -253,5 +253,10 @@ do
   fi
 done
 
-
-exec java $JVM_OPTS -jar $SERVER
+# If we have a gamerules.txt file... feed that in to the server stdin
+if [ -f /data/gamerules.txt ];
```

Comment:
Thanks for contribution.

How about you have it look for something more generic, like `/data/bootstrap.txt`, to expand the semantics of how this be used.